projects/packages/account-protection/src/class-password-detection.php

```diff
@@ -402,6 +402,12 @@ private function user_requires_protection( \WP_User $user, string $password ): b
 			return false;
 		}
 
+		$user_requires_protection = apply_filters( Config::PREFIX . '_user_requires_protection', true, $user );
```

Comment:
```suggestion
		$user_requires_protection = apply_filters( 'account_protection_user_requires_protection', true, $user );
```
I would recommend using a complete, fixed string here, as filters are essentially an API for site owners and other developers, and we don't want to break compatibility/schema if the constant changes. A static string designs for that - what do you think?